app/packages/looker-3d/src/state.ts

```diff
@@ -127,3 +127,15 @@ export const activeNodeAtom = atom<FoSceneNode>({
   key: "fo3d-activeNode",
   default: null,
 });
+
+export const cuboidLabelLineWidthAtom = atom<number>({
```

Comment:
Type is implicit via the default value :)
```suggestion
export const cuboidLabelLineWidthAtom = atom({
```